src/cmake-tools.ts

```diff
@@ -444,7 +498,11 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
   }
 
   async editCache(): Promise<void> {
-    const drv = await this._cmakeDriver;
+    const drv = await this.getCMakeDriverInstance();
+    if (!drv) {
+      return;
```

Comment:
I think this should be returned as a `Promise`:
```ts
return Promise.resolve();
```

Reply:
Style guide question